Wikipedia/Code/DonateFunnel.swift

```diff
@@ -286,4 +290,64 @@ import WMF
 
         logEvent(activeInterface: .articleBanner, action: .impressionSuppressed, actionData: actionData)
     }
+    
+    func logArticleProfile() {
+        logEvent(activeInterface: .articleProfile, action: .profileClick)
+    }
+    
+    func logArticleProfileDonate() {
+        logEvent(activeInterface: .articleProfile, action: .donateStartClick)
+    }
+    
+    func logExploreProfile() {
+        logEvent(activeInterface: .exploreProfile, action: .profileClick)
+    }
+    
+    func logExploreProfileDonate() {
+        logEvent(activeInterface: .exploreProfile, action: .donateStartClick)
+    }
+    
+    func logOptOutExploreProfileDonate() {
+        logEvent(activeInterface: .exploreOptOut, action: .donateStartClick)
+    }
+    
+    func logArticleProfileDonateCancel() {
+        logEvent(activeInterface: .articleProfile, action: .cancelClick)
+    }
+    
+    func logExploreProfileDonateCancel() {
+        logEvent(activeInterface: .exploreProfile, action: .cancelClick)
+    }
+    
+    func logExploreOptOutProfileDonateCancel() {
+        logEvent(activeInterface: .exploreOptOut, action: .cancelClick)
+    }
+    
+    func logExploreProfileDonateApplePay() {
+        logEvent(activeInterface: .exploreProfile, action: .applePayClick)
+    }
+    
+    func logArticleProfileDonateApplePay() {
+        logEvent(activeInterface: .articleProfile, action: .applePayClick)
+    }
+
+    func logExploreOptOutProfileDonateApplePay() {
+        logEvent(activeInterface: .exploreOptOut, action: .applePayClick)
+    }
+    
+    func logExploreProfileDonateWebPay() {
+        logEvent(activeInterface: .exploreProfile, action: .webPayClick)
+    }
+    
+    func logArticleProfileDonateWebPay() {
+        logEvent(activeInterface: .articleProfile, action: .webPayClick)
+    }
+    
+    func logExploreOptOutProfileDonateWebPay() {
+        logEvent(activeInterface: .exploreOptOut, action: .applePayClick)
+    }
+    
+    func logExploreOptOutProfileClick() {
```

Comment:
@l-olson1214 This method needs to be called from somewhere (probably in WMFSettingsViewController.m, userdidTapProfile)